clients/src/main/resources/common/message/ConsumerGroupHeartbeatRequest.json

```diff
@@ -13,6 +13,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//Version 1 added SubscribedTopicRegex to the request for KIP-848
```

Comment:
I would expect you to actually define v1 including the extra field, and also v1 of the response (which I suppose is structurally identical to v0).

Reply:
@AndrewJSchofield Understood, I will change the `subscribedPattern` accordingly